miden-tx/tests/p2id_script_test.rs

```diff
@@ -77,7 +78,7 @@ fn test_p2id_script() {
         .unwrap();
 
     // Execute the transaction and get the witness
-    let transaction_result = executor
+    let executed_transaction = executor
```

Comment:
This is old terminology as `TransactionExecutor` now returns `ExecutedTransaction` rather than `TransactionResult`. So, I'm just fixing these things when I come across them.